### PR TITLE
DAOS-7558 tests: daos_racer for daily_regression and full_regression

### PR DESCRIPTION
**Pull request by Nasf-Fan**
For checking system stability and robustness.

Signed-off-by: Fan Yong <fan.yong@intel.com>

### CODE REVIEW

**Comment by Nasf-Fan** [comment]
set#2 pass CI tests. Restart it for another try.

**Comment by Nasf-Fan** [comment]
set#3 pass CI tests. Restart it for another try:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-5774/3/artifact/Functional%20Hardware%20Large/daos_racer/daos_racer.py/

**Review by jolivier23 — DISMISSED** [review]

**Comment by Nasf-Fan** [comment]
set#4 pass CI tests.

**Review by wangdi1 — DISMISSED** [review]

**Comment by Nasf-Fan** [comment]
@johannlombardi ,what is our plan for daos_racer: keep it as weekly run (full_regression) or daily test (daily_regression) or per-PR?

File: src/tests/ftest/daos_racer/daos_racer.py [hunk 1]
```diff
@@ -29,7 +29,9 @@ def test_daos_racer(self):
         Use Cases:
             Running simultaneous, conflicting I/O requests.
 
-        :avocado: tags=all,full_regression,hw,large,io,daosracer
+        :avocado: tags=all,pr,full_regression,daily_regression
```
Comment (johannlombardi) [inline src/tests/ftest/daos_racer/daos_racer.py:32]: let's remove PR from there.

**Comment by Nasf-Fan** [comment]
Remove PR as suggested.

**Comment by Nasf-Fan** [comment]
Ping @wangdi1 @jolivier23 @johannlombardi , please help to review. Thanks!

**Comment by sylviachanoiyee** [comment]
We probably do not want to enable this test in daily regression because it takes about 2hrs to complete.
It is currently enabled in weekly testing

**Comment by johannlombardi** [comment]
@sylviachanoiyee My understanding is that @Nasf-Fan is reducing the run time in this PR, no?

**Comment by sylviachanoiyee** [comment]
Sure, but do we want to reduce the runtime and run it on both daily and weekly?
I think can should either run it on daily with reduced runtime or just keep it on weekly with longer runtime.

**Comment by Nasf-Fan** [comment]
> Sure, but do we want to reduce the runtime and run it on both daily and weekly?
> I think we should either run it on daily with reduced runtime or just keep it on weekly with longer runtime.

Can we have two versions of daos_racer test: for weekly test, keep 2hrs run time; for daily test, reduce it to 10 minutes or similar?

**Comment by sylviachanoiyee** [comment]
> > Sure, but do we want to reduce the runtime and run it on both daily and weekly?
> > I think we should either run it on daily with reduced runtime or just keep it on weekly with longer runtime.
> 
> Can we have two versions of daos_racer test: for weekly test, keep 2hrs run time; for daily test, reduce it to 10 minutes (as the patch does) or similar?

Yes, we can do that. Basically do a stress racer on weekly and a quick racer on daily. 
However, in order to do that, I think the easiest way is to have a separate py and yaml for the quick racer.
Basically have a daos_racer_quick.py with daily_regression tag like this:
        :avocado: tags=all,daily_regression
        :avocado: tags=hw,large,io,daosracer
and a daos_racer_quick.yaml with runtime 600 and clush_timeout 900
No changes to existing daos_racer.py and daos_racer.yml for full_regression (weekly) test.

**Review by sylviachanoiyee — APPROVED** [review]

**Comment by Nasf-Fan** [comment]
@johannlombardi , do we still need this patch?

**Review by jolivier23 — APPROVED** [review]

**Pull request merged**